tests/test_dialects.py

```diff
@@ -1925,6 +1931,25 @@ def test_spark(self):
             write="spark",
         )
 
+        self.validate(
+            "TO_DATE(x, 'yyyy-MM-dd')",
+            "CAST(DATE_PARSE(SUBSTR(CAST(x AS VARCHAR), 1, 10), '%Y-%m-%d') AS DATE)",
+            read="spark",
+            write="presto",
+        )
+        self.validate(
+            "TO_DATE(x, 'yyyy')",
+            "DATE_PARSE(x, '%Y')",
```

Comment:
These are not the same. `TO_DATE` returns a date while `DATE_PARSE` returns a timestamp.